### PR TITLE
feat(client): diff color based on lock status

### PR DESCRIPTION
**Pull request by someone0002**
Instead of offering how to the normal showTextUI is, I thought it would be good to make it so it's a color based on the lock status, it would a clean change and slick rather than a basic color. Makes it more vibrant rather than basic.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -324,7 +324,27 @@ CreateThread(function()
 
 		if ClosestDoor and ClosestDoor.distance < ClosestDoor.maxDistance then
 			if Config.DrawTextUI and not ClosestDoor.hideUi and ClosestDoor.state ~= showUI then
-				lib.showTextUI(ClosestDoor.state == 0 and lockDoor or unlockDoor)
+				if ClosestDoor.state == 0 then
+					lib.showTextUI("[E] - Lock Door", {
```
Comment (Manason) [inline client/main.lua]: use lockDoor and unlockDoor rather than string literals.

File: client/main.lua [hunk 2]
```diff
@@ -324,7 +324,27 @@ CreateThread(function()
...
+						position = "right-center",
+						icon = 'hand',
+						style = {
+							borderRadius = 0,
+							backgroundColor = '#48BB78',
+							color = 'white'
```
Comment (Manason) [inline client/main.lua]: colors should probably be in config

**Review by Manason — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 2 above)
Comment (someone0002) [reply client/main.lua]: Done.

File: client/main.lua (hunk 1 above)
Comment (someone0002) [reply client/main.lua]: Done.

File: client/main.lua [hunk 3]
```diff
@@ -324,7 +325,29 @@ CreateThread(function()
...
+			    local style = {}
+			    if config.useColors then
+				style.backgroundColor = config.lockColor
+			    end
+			    lib.showTextUI(text, {
+				position = "top-center",
+				icon = 'hand',
+				style = style
+			    })
+			end
```
Comment (LukeWasTakenn) [inline client/main.lua:350]: I'm not sure if this is something we'd want but anyways, why not use ternary instead of copy pasting the code twice?

Comment (someone0002) [reply client/main.lua:350]:
```
local text, style = config[ClosestDoor.state == 0 and "unlockText" or "lockText"], {}
if config.useColors then
    style.backgroundColor = config[ClosestDoor.state == 0 and "unlockColor" or "lockColor"]
end
lib.showTextUI(text, {
    position = "top-center",
    icon = 'hand',
    style = style
})
```

Like so?

**Pull request closed without merging**